packages/react-reconciler/src/ReactFiberPerformanceTrack.js

```diff
@@ -789,88 +803,166 @@ export function logRenderPhase(
   startTime: number,
   endTime: number,
   lanes: Lanes,
+  debugTask: null | ConsoleTask,
 ): void {
   if (supportsUserTiming) {
+    if (endTime <= startTime) {
+      return;
+    }
```

Comment:
I see that you've added this check to the loggers, but what has changed so that you need to check this?

If I remember correctly, DevTools just won't show entries on a timeline, where endTime < startTime, but they would still be present in an exported json-file, which could be useful for debugging Perf tracks implementation.

Reply:
This is unrelated to this PR but it's the same as the change I made here for some other ones:

https://github.com/facebook/react/pull/33659/commits/b3f67f981cc0d921d6fc7c800a1e14944fc14a54

The problem is that they get logged as zero-width entries but those stack on top of whatever other entry is at the same time. Which causes the Scheduler track row to turn into a two row track instead of one and there's this little single pixel lines on the second line.

Ideally that track shouldn't even need to be expanded. It should be clickable while on a single line from the start.